app/controllers/admin/quickbooks_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::QuickbooksController < Admin::AdminController
+  def authenticate
+    authorize :quickbooks, :authenticate?
+
+    redirect_to("https://appcenter.intuit.com/Connect/Begin?oauth_token=#{qb_request_token}")
+  end
+
+  def oauth_callback
+    authorize :quickbooks, :oauth_callback?
+    @header_disabled = true
+
+    Division.root.quickbooks_connect(access_token: qb_access_token, params: params)
```

Comment:
This method name makes it look like the method is doing the connection. But really it's already connected and you're just storing the data. Maybe just call it `quickbooks_save`. And instead of disconnect below you could call it `forget`.